core/components/atoms/collapsible/__stories__/index.story.jsx

```diff
@@ -23,7 +23,7 @@ export const all = () => {
 const customCode = `() => {
   const [expanded, setExpanded] = React.useState(false);
   return (
-    <div style={{height: "100vh"}}>
+    <div className="d-flex">
```

Comment:
Please use `vh-100` class instead of `d-flex`.
You can refer [here](https://mds.innovaccer.com/?path=/story/others-styles-sizing--sizing)